### PR TITLE
feat(structure): update TimelineItem to match new UI, prepare for chunks

### PR DESCRIPTION
**Pull request by pedrobonamin**
### Description

Updates the TimelineItem component to match the new UI.
It also adds placeholders for chunks , collaborators and menu.

A story has been added showing the changes for easier review and testing, you can find it here https://test-studio-git-corel-144.sanity.dev/test/workshop/structure;panes;documentTimelineMenu;timelineItem?scheme=light

Screenshot 2024-08-30 at 16 38 24
Screenshot 2024-08-30 at 16 38 53

### What to review

Open a document history, does it looks as expected?

### Testing
No new testing has been added, existing tests cover this functionality, when moving further, e2e tests will be included to validate the history squash

### Notes for release
n/a

### CODE REVIEW

File: packages/sanity/src/structure/i18n/resources.ts [hunk 1]
```diff
@@ -448,6 +448,13 @@ const structureLocaleStrings = defineLocalesResources('structure', {
   'structure-error.reload-button.text': 'Reload',
   /** Labels the structure path of the structure error screen */
   'structure-error.structure-path.label': 'Structure path',
+
+  /** The aria label for the menu button in the timeline item */
+  'timeline-item.menu-button.aria-label': 'Open action menu',
+  /** The text for the tooltip in menu button the timeline item */
+  'timeline-item.menu-button.tooltip': 'Actions',
+  /** The text for the expand action in the timeline item menu */
+  'timeline-item.menu.action-expand': 'Expand',
```
Comment (pedrobonamin) [inline packages/sanity/src/structure/i18n/resources.ts:457]: This are placeholder, text value is not definitive.

Comment (RitaDias) [reply packages/sanity/src/structure/i18n/resources.ts:457]: Might worth adding a `todo` in the code so it's easier to find :)

File: packages/sanity/src/structure/panes/document/timeline/timelineItem.tsx
Comment (RitaDias) [inline packages/sanity/src/structure/panes/document/timeline/timelineItem.tsx:1]: This looks so much nicer!

File: packages/sanity/src/structure/panes/document/timeline/timelineItem.tsx [hunk 2]
```diff
@@ -67,59 +131,42 @@ export function TimelineItem({
...
-      data-testid="timeline-item-button"
-      data-chunk-id={chunk.id}
-      data-first={isFirst ? true : undefined}
-      data-last={isLast ? true : undefined}
-      data-ui="timelineItem"
-      mode={isSelected ? 'default' : 'bleed'}
-      onClick={handleClick}
-      padding={0}
-      radius={2}
-      tone={isSelected ? 'primary' : TIMELINE_ITEM_EVENT_TONE[chunk.type]}
```
Comment (RitaDias) [inline packages/sanity/src/structure/panes/document/timeline/timelineItem.tsx:82]: Just to double check, we're not breaking some functionality with removing this right? 😬

**Review by RitaDias — APPROVED** [review]
Let's goooo, fantastic work!
I left a comment because removing a lot of props from a element make me a bit nervous but everything worked as intended

File: packages/sanity/src/structure/panes/document/timeline/timelineItem.tsx (hunk 2 above)
Comment (pedrobonamin) [reply packages/sanity/src/structure/panes/document/timeline/timelineItem.tsx:82]: I think it should not affect any existing functionality, all of this props received in the element were used for styling.
I'm keeping this ones given they are used for ui and testing. 
```typescript
   data-ui="timelineItem"
   data-testid="timeline-item-button"
   data-chunk-id={chunk.id}
```

**Pull request merged**